<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionWindow</class>
 <widget class="QDialog" name="SelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnAvatar">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogout">
   <property name="geometry">
    <rect>
     <x>549</x>
     <y>427</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar sesión</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRandom">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>¡Problema aleatorio!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnToggleList">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver lista de problemas ▼</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listProblems">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>231</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
